1_space_odyssey/js/mannuel-9000/app.js

```diff
@@ -114,9 +114,18 @@ console.log(name, isAlive, hasJetPack, eyes, eyeColor, hairColor, legs, arms, ha
        cannot go outside, console log the correct log based on that condition. 
 */
 
+
+
+if (hasJetPack){
+console.log("JetPack is enabled!!! off I go outside to fix the satellite");
+}
+else if (jetpacknotenabled){
```

Comment:
if you are only checking one condition or one boolean value, if...else is good enough.

```JavaScript
if (someThing) {
  ...
} else {
 ...
}
```

if you are checking more than one condition then you can use if...else if...else if, and then if there are too many use a switch statement.

![https://media.giphy.com/media/13SCbeE9AD5s2c/giphy-downsized.gif](https://media.giphy.com/media/13SCbeE9AD5s2c/giphy-downsized.gif)

Reply:
you are only checking 1 condition, so you do not need the `else if (jetpacknotenabled){`